intellij-toml/src/test/kotlin/org/toml/ide/formatter/TomlFormatterTest.kt

```diff
@@ -0,0 +1,10 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.toml.ide.formatter
+
+class TomlFormatterTest : TomlFormatterTestBase() {
+    fun `test spacing`() = doTest()
```

Comment:
Let's better inline test texts.
I find it much easier to have code example in test class directly instead of looking for before/after files in test data